### DISCUSSION TITLE
When I use the compositon api, how should I use the nextTick method, do I need to introduce vue?

### DISCUSSION THREAD

**Discussion by senher** [nuxt/nuxt#17004] [Questions] [answered]
When I use the compositon api, how should I use the nextTick method, do I need to introduce vue?
import { nextTick } from 'vue'

**Comment by viii47** [comment, marked as answer]
nextTick can be used without any imports, simply `nextTick(() => { ... }`

**Reply by senher** [reply]
Hi~,I use swiper in the page and call int swiper in nextick, but it doesn't work (jump via nuxt-link);
`setTimeout(() => {
    initSwiper();
  }, 2000);`
If I put init in setTimeout it works fine;

`initSwiper();`
and if I refresh the page directly it will work fine( use init swiper in setup do not need nextTick() );
